src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -71,44 +69,41 @@ private static void mapExample(HazelcastInstance client) {
      * @param client - a {@link HazelcastInstance} client.
      */
     private static void sqlExample(HazelcastInstance client) {
-
         System.out.println("Creating a mapping...");
-        // See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+        // See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
         final String mappingQuery = ""
-            + "CREATE OR REPLACE MAPPING cities TYPE IMap"
-            + " OPTIONS ("
-            + "     'keyFormat' = 'java',"
-            + "     'keyJavaClass' = 'java.lang.String',"
-            + "     'valueFormat' = 'java',"
-            + "     'valueJavaClass' = 'java.lang.String'"
-            + " )";
+                + "CREATE OR REPLACE MAPPING capitals TYPE IMap"
+                + " OPTIONS ("
+                + "     'keyFormat' = 'varchar',"
+                + "     'valueFormat' = 'varchar'"
+                + " )";
         try (SqlResult ignored = client.getSql().execute(mappingQuery)) {
             System.out.println("The mapping has been created successfully.");
         }
         System.out.println("--------------------");
 
         System.out.println("Deleting data via SQL...");
-        try (SqlResult ignored = client.getSql().execute("DELETE FROM cities")) {
+        try (SqlResult ignored = client.getSql().execute("DELETE FROM capitals")) {
             System.out.println("The data has been deleted successfully.");
         }
         System.out.println("--------------------");
-        
+
         System.out.println("Inserting data via SQL...");
         String insertQuery = ""
-            + "INSERT INTO cities VALUES"
-            + "('Australia','Canberra'),"
-            + "('Croatia','Zagreb'),"
-            + "('Czech Republic','Prague'),"
-            + "('England','London'),"
-            + "('Turkey','Ankara'),"
-            + "('United States','Washington, DC');";
+                + "INSERT INTO capitals VALUES"
+                + "('Australia','Canberra'),"
+                + "('Croatia','Zagreb'),"
+                + "('Czech Republic','Prague'),"
+                + "('England','London'),"
+                + "('Turkey','Ankara'),"
+                + "('United States','Washington, DC');";
         try (SqlResult ignored = client.getSql().execute(insertQuery)) {
             System.out.println("The data has been inserted successfully.");
         }
         System.out.println("--------------------");
 
         System.out.println("Retrieving all the data via SQL...");
-        try (SqlResult result = client.getSql().execute("SELECT * FROM cities")) {
+        try (SqlResult result = client.getSql().execute("SELECT * FROM capitals")) {
 
             for (SqlRow row : result) {
                 String country = row.getObject(0);
```

Comment:
```suggestion
                String country = row.getObject("country");
```
If you add the column names in the mapping, we can use actual names here instead of indexes and it may be better visually for the user.

Reply:
i think it was like this to show both ways to use it. there are usages with actual names in the other code samples. i think it is better this way